### DISCUSSION TITLE
how highlight multiple code lines

### DISCUSSION THREAD

**Discussion by zxkane** [chipzoller/hugo-clarity#289] [Q&A] [answered]
I want to highlight multiple lines(>20) of code, just found below syntax working,

```
```yaml {hl_lines=[6,7,8,9,10]}
blabla...
```

is there any way to highlight 6~30 lines with a simpler syntax?

**Comment by chipzoller** [comment, marked as answer]
I believe a dash is supported as a range operator.

**Reply by zxkane** [reply]
Any example for it? Tried it like below, it does not work.

```
```yaml {hl_lines=[6-20]}
```

**Reply by chipzoller** [reply]
`{hl_lines=["6-20"]}`

**Reply by zxkane** [reply]
It works! Might document it for further reference.

**Reply by chipzoller** [reply]
Done in https://github.com/chipzoller/hugo-clarity/commit/3fcb1db2eaddee13cd5c9b1a69e7e644edf079c8
